### PR TITLE
[LA64_DYNAREC] Added more opcodes and fixed more things

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

File: src/dynarec/la64/dynarec_la64_00.c [hunk 1]
```diff
@@ -193,6 +209,18 @@ uintptr_t dynarec64_00(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 SMWRITELOCK(lock);
             }
             break;
+        case 0x8B:
+            INST_NAME("MOV Gd, Ed");
+            nextop=F8;
+            GETGD;
+            if(MODREG) {
+                MVxw(gd, xRAX+(nextop&7)+(rex.b<<3));
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_00.c:217]: Probably need `TO_LA64`macro here

Comment (ksco) [reply src/dynarec/la64/dynarec_la64_00.c:217]: Ah yes, sorry

Comment (ksco) [reply src/dynarec/la64/dynarec_la64_00.c:217]: It's a bit annoying the register number can continuous.

Comment (ptitSeb) [reply src/dynarec/la64/dynarec_la64_00.c:217]: An alternative way could be also: `(nextop&7)+((rex.b<<3)?xR8:xRAX)` not sure it's usefull, but that can be a possibility.

Comment (ptitSeb) [reply src/dynarec/la64/dynarec_la64_00.c:217]: > It's a bit annoying the register number cannot be continuous.

I agree, but there is no choice :(

File: src/dynarec/la64/dynarec_la64_emit_math.c [hunk 2]
```diff
@@ -171,9 +170,8 @@ void emit_add32c(dynarec_la64_t* dyn, int ninst, rex_t rex, int s1, int64_t c, i
     IFX(X_CF)
     {
         if (rex.w) {
-            MOV32w(x2, 0xffffffff);
```
Comment (ptitSeb) [inline src/dynarec/la64/dynarec_la64_emit_math.c:174]: oh, yeah, that was just wrong before. mixing `x2` direct access register with `s1`/`s2` arbitrary scratch register is a sure fail at some point!

**Pull request merged**